tests/Unit/NumericalAlgorithms/Spectral/Test_ChebyshevGauss.cpp

```diff
@@ -0,0 +1,81 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <cstddef>
+#include <utility>
+
+#include "DataStructures/DataVector.hpp"
+#include "NumericalAlgorithms/Spectral/Spectral.hpp"
+
+namespace Spectral {
+// To test the \f$w_k\f$ as opposed to the Spectral::quadrature_weights. We
```

Comment:
remove `\f$` as this is not a doxygen comment

Reply:
I'd prefer to mark up comments the same as doxygen comments for consistency, this also makes it easier to move comments around between doxygen-formatted environments and others. If you're strongly opposed I'll change it though.

Reply:
Not sure what’s best practice here. I follow the Doxygen documentation where they don’t put types and functions in backticks because they are parsed and linked: https://www.stack.nl/~dimitri/doxygen/manual/autolink.html

Reply:
We use backticks literally everywhere in the Doxygen documentation...